terraform/modules/prometheus/resources.tf

```diff
@@ -8,6 +8,7 @@ resource cloudfoundry_app prometheus {
   name             = "prometheus-${var.name}"
   space            = var.space_id
   path             = data.archive_file.config.output_path
+  memory           = "5120"
```

Comment:
Could we make it a module parameter, with default 1G?
Also can you try to enter it as integer instead of string? ie `5120` vs `"5120"`